data/101/815/073/101815073.geojson

```diff
@@ -67,6 +70,9 @@
     "name:und_x_variant":[
         "Jakobi"
     ],
+    "name:unk_x_variant":[
+        "Jakobi"
```

Comment:
Same question about why this unknown name variant is coming from.

Reply:
https://github.com/whosonfirst-data/whosonfirst-data-admin-ee/pull/11#discussion_r463332361, same applies here